utils/tang/patches/010-run-standalone.patch

```diff
@@ -0,0 +1,912 @@
+diff --git a/meson.build b/meson.build
```

Comment:
This is a very large patch - I assume this is https://github.com/latchset/tang/pull/86 but there is no patch comment to explain.

I think it would be best to see some indication of upstream accepting this patch before merging here.

Reply:
That's my position too.